spec/hutch/worker_spec.rb

```diff
@@ -45,9 +45,17 @@
     end
 
     it 'sets up a subscription' do
-      expect(queue).to receive(:subscribe).with(manual_ack: true)
+      expect(queue).to receive(:subscribe).with(consumer_tag: %r(^hutch\-.{36}$), manual_ack: true)
```

Comment:
Seems like it's 255 bytes max: https://github.com/rabbitmq/rabbitmq-common/blob/rabbitmq_v3_6_0/codegen.py#L178-L179 ... I hope that's the same in 3.4.x as well :)